### PR TITLE
Paniukhin Nikita

### PR DESCRIPTION
**Pull request by npanuhin**

### CODE REVIEW

File: Main.hs [hunk 1]
```diff
@@ -4,22 +4,47 @@ import Control.Monad (unless)
...
+short _         = True
 
 lovely :: [Int] -> Bool
-lovely = undefined 
+lovely (_:_:14:_) = True
+lovely xs         = short xs
 
 rightTriangles :: [(Int, Int, Int)]
-rightTriangles = undefined 
+rightTriangles = [(a, b, c) | c <- [1..], a <- reverse [1..c], b <- [a..c], a^2 + b^2 == c^2]
```
Comment (alex28sh) [inline Main.hs:15]: What is the point of making excessive reverse call?

Comment (alex28sh) [reply Main.hs:15]: try swapping a and b in comprehension)

Comment (npanuhin) [reply Main.hs]: [Fixed](https://github.com/kajigor/fp-2024-cub-fall/pull/1/commits/c0f2c95b320b069855d4d973b7dbc59d09824b86#diff-894c9c531eb0d7c5f20f08d4008d2dfe4e48d3e542af618ed58a9b281cd28aa3), thank you!

File: Main.hs [hunk 2]
```diff
@@ -4,22 +4,47 @@ import Control.Monad (unless)
...
+ageOn planet ageInSeconds
+  | planet == "Mercury" = ageInSeconds / secondsPerEarthYear / 0.2408467
+  | planet == "Venus"   = ageInSeconds / secondsPerEarthYear / 0.61519726
+  | planet == "Earth"   = ageInSeconds / secondsPerEarthYear
+  | planet == "Mars"    = ageInSeconds / secondsPerEarthYear / 1.8808158
+  | planet == "Jupiter" = ageInSeconds / secondsPerEarthYear / 11.862615
+  | planet == "Saturn"  = ageInSeconds / secondsPerEarthYear / 29.447498
+  | planet == "Uranus"  = ageInSeconds / secondsPerEarthYear / 84.016846
+  | planet == "Neptune" = ageInSeconds / secondsPerEarthYear / 164.79132
+  | planet == "Pluto"   = error "Pluto is not a planet"
```
Comment (kajigor) [inline Main.hs:36]: You have code duplication here. It's better to introduce a function for it.

**Comment by kajigor** [comment]
ok

**Pull request closed without merging**